tsdb/index/postings.go

```diff
@@ -168,12 +168,12 @@ func (p *MemPostings) EnsureOrder() {
 	wg.Add(n)
 
 	for i := 0; i < n; i++ {
-		go func() {
+		go func(i interface{}) {
```

Comment:
```suggestion
		go func() {
```
That will need to be reverted